src/components/StatisticsGrid/style.module.scss

```diff
@@ -3,7 +3,7 @@
   display: grid;
```

Comment:
![Screen Shot 2022-10-12 at 11 26 05 AM](https://user-images.githubusercontent.com/33165426/195419738-bea94a56-26db-47ca-bafd-ee759b37a06f.jpg)

![Screen Shot 2022-10-12 at 11 26 16 AM](https://user-images.githubusercontent.com/33165426/195419770-4c3c786f-368a-41cc-8310-f7e60e360799.jpg)

Try to always keep content contained within the container it's defined in so we avoid this possibility of overlapping neighbors and content overflow. You can handle this either by shrinking the content on smaller screen sizes or having the container grow to always fit the items inside (although this might create inconsistent grid container sizes)

![Screen Shot 2022-10-12 at 11 29 23 AM](https://user-images.githubusercontent.com/33165426/195420375-168ada6c-fe59-4502-bcca-93d4bb798b6c.jpg)
Also, like I mentioned in the previous comment, try to get the UI to function the way you want until 425px. You could potentially reduce the width of the grid items or try to center it better so it doesn't misalign to the left completely.